### PR TITLE
Set up maestro insertions for dotnet/core-sdk

### PR DESCRIPTION
**Pull request by dsplaisted**
This should enable maestro insertions of .NET Core runtime components into dotnet/core-sdk.

@natemcmaster @mmitche This doesn't currently cover ASP.NET.  What would we need to add to this to get ASP.NET updates via maestro?  I see this for the 2.2.1xx branch:

```json
    {
      "triggerPaths": [
        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/release/2.2/packages.semaphore",
      ],
      "action": "cli-dependencies",
      "delay": "00:05:00",
      "actionArguments": {
        "vsoSourceBranch": "release/2.2.1xx",
        "vsoBuildParameters": {
          "GITHUB_PULL_REQUEST_NOTIFICATIONS": "dotnet/dotnet-cli",
          "GITHUB_UPSTREAM_BRANCH": "release/2.2.1xx",
          // Enable once this is done https://github.com/aspnet/Universe/issues/1013.
          // "ASPNET_VERSION_FRAGMENT": "<version fragment>",
          "SDK_VERSION_FRAGMENT": "dotnet/sdk/release/2.2.1xx",
          "TEMPLATING_VERSION_FRAGMENT": "dotnet/templating/release/2.2",
          "WEBSDK_VERSION_FRAGMENT": "dotnet/templating/release/2.2.1xx",
          "CORESETUP_VERSION_FRAGMENT": "dotnet/core-setup/release/2.2"
        }
      }
    },
```

As I understand it, that trigger is for a ProdCon build, which we don't have yet for 3.0.  Is there a trigger that would work for ASP.NET for 3.0 currently?

### CODE REVIEW

**Comment by dsplaisted** [comment]
@eerhardt @mmitche @nguerrera for review

**Comment by nguerrera** [comment]
I'm surprised there's no lines deleted. Is whatever was attempting 
https://github.com/dotnet/cli/pull/9539 now gone?

**Comment by mmitche** [comment]
@dsplaisted What we need is for aspnet to publish to the versions repo upon build completion.  In prodcon this is done automatically (or should be as of the next build).  @natemcmaster may know more.

/fyi @jamshedd

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -102,6 +108,12 @@
       "vsoProject": "DevDiv",
       "buildDefinitionId": 7644
     },
+    // A build definition that will update the dependencies of the dotnet/toolset repo
```
Comment (eerhardt) [inline Maestro/subscriptions.json]: I assume this will be used in the future?  I don't see it used here.

**Comment by mmitche** [comment]
@nguerrera Is right.  I think this should be deleted:

https://github.com/dotnet/versions/blob/d770e43f929233f18e53721ad92942cc1c1f8c07/Maestro/subscriptions.json#L1366

and the roslyn fragment should be added for the appropriate branch if applicable: https://github.com/dotnet/versions/blob/master/Maestro/subscriptions.json#L1377

**Comment by natemcmaster** [comment]
@mmitche is right. aspnet doesn't push updates to dotnet/versions. The only way to trigger updates with maestro right now is by triggering off prodcon updates.

For example, here is a trigger for 2.1 prodcon builds. This runs a script in which makes a PR to update aspnet/Universe to versions from the latest prodcon output.

https://github.com/dotnet/versions/blob/7632f6765fdddf240bb266ba5c89c51a4a2f5dda/Maestro/subscriptions.json#L1279-L1299

File: Maestro/subscriptions.json (hunk 1 above)
Comment (dsplaisted) [reply Maestro/subscriptions.json]: Yes, I'm still working on updating the update-dependencies process in dotnet/toolset.  When that is done then we can add a subscription here.

**Pull request merged**